examples/yacat/yacat.py

```diff
@@ -20,146 +24,184 @@
     TEXT_COLOR_YELLOW,
 )
 
+HASHCAT_ATTACK_MODE = 3  # stands for mask attack, hashcat -a option
+KEYSPACE_OUTPUT_PATH = Path("/golem/output/keyspace")
 
-def write_hash(hash):
-    with open("in.hash", "w") as f:
-        f.write(hash)
+# Ideally, this value should depend on the chunk size
+ATTACK_TIMEOUT: timedelta = timedelta(minutes=30)
+KEYSPACE_TIMEOUT: timedelta = timedelta(minutes=10)
 
+arg_parser = build_parser("Run a hashcat attack (mask mode) on Golem network")
+arg_parser.add_argument("--hash", type=str, help="Target hash to be cracked", required=True)
+arg_parser.add_argument(
+    "--mask", type=str, help="Hashcat mask to be used for the attack", required=True
```

Comment:
I'd provide an example value for `--mask` somewhere, preferably in `help` strings in the parser, to make this example runnable more easily without referring to external docs (apart from general golem/`yapapi` docs)

Reply:
Or maybe provide an example invocation in the string passed to `build_parser()`?

Reply:
Good one! Actually I might just do both. :shrug: